src/language-js/parser-babel.js

```diff
@@ -58,13 +62,42 @@ function resolvePluginsConflict(
   return combinations;
 }
 
+// https://github.com/babel/babel/pull/7934/files#diff-a739835084910b0ee3ea649df5a4d223R67
+const FLOW_PRAGMA_REGEX = /\*?\s*@(?:no)?flow\b/;
```

Comment:
```suggestion
const FLOW_PRAGMA_REGEX = /@(?:no)?flow\b/;
```
This part at the beginning doesn't make any difference.

Reply:
Right, how  about `(?:^|[*\s])`

Reply:
We don't need anything in front of `@` to align with Babel. Babel is okay even with `// wolf@flow`.

Reply:
You are right, there is many problems in this regex. `someone@flow.com` 😄

Reply:
Flow might recognize those too.

Reply:
Tested it with Flow. Flow doesn't recognize `wolf@flow`, `+@flow` and even `@flow@` as pragmas. On the other hand, it recognizes `// /@flow` and `// *@flow`. Anyway, let's stay consistent with Babel.